worlds/shivers/__init__.py

```diff
@@ -1,11 +1,13 @@
-from typing import List
-from .Items import item_table, ShiversItem
-from .Rules import set_rules
-from BaseClasses import Item, Tutorial, Region, Location
+from typing import Dict, List
+from typing import Optional
```

Comment:
You can combine these
```suggestion
from typing import Dict, List, Optional
```

Reply:
Thought I ran import optimization on the Shivers directory